presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -143,15 +143,15 @@ private static long epochDaysInZone(TimeZoneKey timeZoneKey, long instant)
     public void testLocalTime()
             throws Exception
     {
-        long millis = new LocalTime(session.getStartTime(), DATE_TIME_ZONE).getMillisOfDay();
+        long millis = new LocalTime(session.getStartTime(), DateTimeZone.UTC).getMillisOfDay();
```

Comment:
I wonder if there is a better way to write this test. It doesn't need to be in this commit or this PR.